setup.py

```diff
@@ -103,7 +103,8 @@ def run(self):
             'flocker-deploy = flocker.cli.script:flocker_deploy_main',
             'flocker-changestate = flocker.node.script:flocker_changestate_main',
             'flocker-reportstate = flocker.node.script:flocker_reportstate_main',
-            'flocker-serve = flocker.node.script:flocker_serve_main',
+            'flocker-volume = flocker.node.script:flocker_volume_main',
```

Comment:
If the main function has its name adjusted, possibly the command line script name should be adjusted correspondingly.